### PR TITLE
83 test overwrite stats

### PR DESCRIPTION
**Pull request by emlynjdavies**
addresses #83 by testing silcam_process with different combinations of overwriteSTATS and checking for expected output

### CODE REVIEW

File: pysilcam/tests/test_overwrite_stats.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+from pysilcam.__main__ import silcam_process
+from pysilcam.silcreport import silcam_report
+from pysilcam.config import load_config
+from pysilcam.postprocess import count_images_in_stats
+from pysilcam.config import PySilcamSettings
+import glob
+import os
+import unittest
+import pandas as pd
+import tempfile
```
Comment (tankefugl) [inline pysilcam/tests/test_overwrite_stats.py:11]: F401: 'tempfile' imported but unused

File: pysilcam/tests/test_overwrite_stats.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+from pysilcam.__main__ import silcam_process
+from pysilcam.silcreport import silcam_report
+from pysilcam.config import load_config
+from pysilcam.postprocess import count_images_in_stats
+from pysilcam.config import PySilcamSettings
+import glob
```
Comment (tankefugl) [inline pysilcam/tests/test_overwrite_stats.py:7]: F401: 'glob' imported but unused

File: pysilcam/tests/test_overwrite_stats.py [hunk 3]
```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+from pysilcam.__main__ import silcam_process
+from pysilcam.silcreport import silcam_report
+from pysilcam.config import load_config
+from pysilcam.postprocess import count_images_in_stats
+from pysilcam.config import PySilcamSettings
```
Comment (tankefugl) [inline pysilcam/tests/test_overwrite_stats.py:6]: F401: 'pysilcam.config.PySilcamSettings' imported but unused

File: pysilcam/tests/test_overwrite_stats.py [hunk 4]
```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+from pysilcam.__main__ import silcam_process
+from pysilcam.silcreport import silcam_report
```
Comment (tankefugl) [inline pysilcam/tests/test_overwrite_stats.py:3]: F401: 'pysilcam.silcreport.silcam_report' imported but unused

**Review by tankefugl — CHANGES_REQUESTED** [review]
[Lintly](https://github.com/grantmcconnaughey/Lintly) has detected code quality issues in this pull request.

File: pysilcam/tests/test_overwrite_stats.py [hunk 5]
```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+from pysilcam.__main__ import silcam_process
+from pysilcam.config import load_config
+from pysilcam.postprocess import count_images_in_stats
+import os
+import unittest
+import pandas as pd
+import tempfile
```
Comment (tankefugl) [inline pysilcam/tests/test_overwrite_stats.py:8]: F401: 'tempfile' imported but unused

**Review by tankefugl — DISMISSED** [review]
[Lintly](https://github.com/grantmcconnaughey/Lintly) has detected code quality issues in this pull request.

**Comment by emlynjdavies** [comment]
merged into #96

**Pull request closed without merging**